core/src/main/java/tachyon/worker/TachyonWorker.java

```diff
@@ -55,41 +55,39 @@
 
   /**
    * Create a new TachyonWorker
-   *
-   * @param masterAddress The TachyonMaster's address
-   * @param workerAddress This TachyonWorker's address
+   * 
+   * @param masterAddress The TachyonMaster's address. e.g., localhost
+   * @param masterPort The TachyonMaster's port. e.g., 19998
+   * @param workerPort This TachyonWorker's port. e.g., 29998
    * @param dataPort This TachyonWorker's data server's port
    * @param minWorkerThreads The min number of worker threads used in TThreadPoolServer
    * @param maxWorkerThreads The max number of worker threads used in TThreadPoolServer
    * @param tachyonConf The instance of {@link tachyon.conf.TachyonConf} to used by Worker.
    * @return The new TachyonWorker
    */
-  public static synchronized TachyonWorker createWorker(InetSocketAddress masterAddress,
-      InetSocketAddress workerAddress, int dataPort, int minWorkerThreads, int maxWorkerThreads,
+  public static synchronized TachyonWorker createWorker(String masterAddress, int masterPort,
+      int workerPort, int dataPort, int minWorkerThreads, int maxWorkerThreads,
       TachyonConf tachyonConf) {
```

Comment:
In the PR description, it says "this change makes Tachyon always create worker through createWorker(TachyonConf conf) API.". Then, why do we still need this API?